<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testUI</class>
 <widget class="QWidget" name="testUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="testButton1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test 2</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>511</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
